src/EventTeachingService.cpp

```diff
@@ -68,7 +68,6 @@ Processed EventTeachingService::handleMessage(unsigned int opc, CANFrame *msg)
 
       if (index < module_config->EE_MAX_EVENTS)
       {
-        // TODO: Review this. j is always 0.
```

Comment:
This was a TODO I added in Duncan's code. This has then been fixed an "j" has been replaced with "index" and properly set.